pandas/_libs/__init__.py

```diff
@@ -1,6 +1,15 @@
-# flake8: noqa
+__all__ = [
```

Comment:
Is this required? I think unnecessary since we don't hide or disallow importing anything that appears here globally

Reply:
The ```__all__``` is required in order to remove the ```# flake8: noqa``` comment, otherwise ```flake8``` complains about unused imports.

Reply:
But not  a deal breaker for me, I only made this change because opening a PR just for the ```LooseVersion``` seemed too small.

Reply:
Ah forgot about the flake8 warning; I think this is OK as well